### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by loganwillsaveyou**
i farted

### CODE REVIEW

**Comment by Star-Cyber3** [comment]
bruh

**Comment by loganwillsaveyou** [comment]
dont blame me

**Comment by Star-Cyber3** [comment]
what
come on

**Comment by Star-Cyber3** [comment]
## hi

**Comment by loganwillsaveyou** [comment]
I SAID DONT BLAME MEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

File: index.html [hunk 1]
```diff
@@ -1 +1,3 @@
-
+<!doctype>
```
Comment (Star-Cyber3) [inline index.html:1]:
```suggestion
<!DOCTYPE html>
```

File: index.html [hunk 2]
```diff
@@ -1 +1,3 @@
-
+<!DOCTYPE html>
+<head>
```
Comment (Star-Cyber3) [inline index.html:2]:
```suggestion
<html>
```

**Review by Star-Cyber3 — APPROVED** [review]
Its good

**Pull request merged**